README.md

```diff
@@ -36,10 +42,13 @@ In Erlang you can use this queries like functions:
 > {ok, Queries} = eql:compile("queries/user.sql"). % with path to your queries file
 > {ok, Q1} = eql:get_query(get_all_users, Queries).
 > {ok, Q2} = proplists:get_value(get_user_by_id, Queries).
+> {ok, Q3} = eql:get_query(get_by_id, Queries, [{table, "some_table"}]).
 > Q1.
-%> "SELECT * FROM users;"
+%> <<"SELECT * FROM users;">>
```

Comment:
The `ok` is matched out in the line above, but the `Q3` is wrong because it kept the `{ok, ...}` so I'm fixing that :)